proxy/http/http.go

```diff
@@ -135,6 +139,7 @@ func (r *Request) WriteHeaderTo(writer *bufio.Writer) error {
 	//read & write the headers
 	return copyHeader(&r.header, r.reader, writer,
 		func(rawHeader []byte) {
+			r.readSize += len(rawHeader)
```

Comment:
not exactly the incoming header traffic, cause the header has been translated and modified